sktime/performance_metrics/forecasting/_classes.py

```diff
@@ -1888,6 +1888,178 @@ def get_test_params(cls, parameter_set="default"):
         return [params1, params2]
 
 
+class MeanSquaredErrorPercentage(BaseForecastingErrorMetricFunc):
+    """
+    Mean Squared Error Percentage (MSE%) forecasting error metric.
+
+    Calculates the mean squared error percentage between the true and predicted values.
+    Optionally, the root mean squared error percentage (RMSE%) can be computed by
+    setting `square_root=True`.
+
+    Parameters
+    ----------
+    square_root : bool, default = False
+        Whether to take the square root of the metric
+    multioutput : {'raw_values', 'uniform_average'}  or array-like of shape \
+            (n_outputs,), default='uniform_average'
+        Defines how to aggregate metric for multivariate (multioutput) data.
+        If array-like, values used as weights to average the errors.
+        If 'raw_values', returns a full set of errors in case of multioutput input.
+        If 'uniform_average', errors of all outputs are averaged with uniform weight.
+    multilevel : {'raw_values', 'uniform_average', 'uniform_average_time'}
+        Defines how to aggregate metric for hierarchical data (with levels).
+        If 'uniform_average' (default), errors are mean-averaged across levels.
+        If 'uniform_average_time', metric is applied to all data, ignoring level index.
+        If 'raw_values', does not average errors across levels, hierarchy is retained.
+    """
+
+    def __init__(
+        self,
+        multioutput="uniform_average",
+        multilevel="uniform_average",
+        square_root=False,
+    ):
+        self.square_root = square_root
+        self.multioutput = multioutput
+
+        super().__init__(multioutput=multioutput, multilevel=multilevel)
+
+    def _evaluate(self, y_true, y_pred, **kwargs):
+        r"""
+        Evaluate the Mean Squared Error Percentage between `y_true` and `y_pred`.
+
+        Parameters
+        ----------
+        y_true : pd.Series or pd.DataFrame
+            Ground truth (actual) target values.
+            Can be a Series or DataFrame for univariate or multivariate forecasts.
+
+        y_pred : pd.Series or pd.DataFrame
+            Forecasted target values.
+            Must have the same shape as `y_true`.
+
+        Returns
+        -------
+        loss : float or pd.Series
+            The calculated Mean Squared Error Percentage.
+            - If `multioutput='raw_values'`, returns a Series with the MSPE for
+            each output.
+            - Otherwise, returns a scalar value representing the aggregated MSPE.
+
+        Notes
```

Comment:
the notes from the private docstrings will not be displayed to the user or the online reference. Explanation should be in the class docstring, this ends up in the API reference (if you add the estimator to `api_reference/performance_metrics.rst`).

Can you kindly move the explanation up there?